pkg/storage/loaders/key_contact.go

```diff
@@ -0,0 +1,68 @@
+package loaders
+
+import (
+	"context"
+
+	"github.com/google/uuid"
+
+	"github.com/cms-enterprise/mint-app/pkg/appcontext"
+	"github.com/cms-enterprise/mint-app/pkg/models"
+	"github.com/cms-enterprise/mint-app/pkg/storage"
+
+	"github.com/graph-gophers/dataloader/v7"
+)
+
+// keyContactLoaders is a struct that holds LoaderWrappers related to Key Contacts
+type keyContactLoaders struct {
+	// ByID returns a single KeyContact by its ID
+	ByID LoaderWrapper[uuid.UUID, *models.KeyContact]
+
+	// GetAll returns all KeyContacts
+	GetAll LoaderWrapper[*uuid.UUID, []*models.KeyContact]
```

Comment:
Not sure this is needed. The point of a data loader is so that the store is only called once. If that's the intent here, I'd suggest we combine with the ByID call.